### ISSUE TITLE
Crash - Laucher can not load the mod.

### ISSUE THREAD

**Issue by ghost** [Hunternif/AntiqueAtlas#240]
Hello

I'll began update my modpack to minecraft 1.14.4
The following link will show you the crash report.

https://paste.ee/p/kCtkj

<script src="https://paste.ee/e/kCtkj/0"></script>

I hope you fix and update this fast.

Thanks

**Comment by TehNut** [comment]
You need to install Fabric API

**Comment by G93d14** [comment]
> You need to install Fabric API

I have fabric API installed, and MultiMC still crashes with Antique Atlas.


[crash-2019-08-27_14.56.41-client.txt](https://github.com/AntiqueAtlasTeam/AntiqueAtlas/files/3549080/crash-2019-08-27_14.56.41-client.txt)
[crash-2019-08-27_14.57.16-client.txt](https://github.com/AntiqueAtlasTeam/AntiqueAtlas/files/3549081/crash-2019-08-27_14.57.16-client.txt)

**Comment by G93d14** [comment]
Here's a fix that worked for me:

The [Cotton](https://www.curseforge.com/minecraft/mc-mods/cotton) toolkit has to be installed with this.

**Comment by Martmists-GH** [comment]
Fabric API and Cotton installed, still getting 
```java
        Suppressed: java.lang.NoClassDefFoundError: io/github/cottonmc/cotton/config/ConfigManager
		at hunternif.mc.atlas.SettingsConfig.loadConfig(SettingsConfig.java:15)
		at hunternif.mc.atlas.AntiqueAtlasMod.onInitialize(AntiqueAtlasMod.java:65)
		at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.lambda$logErrors$0(EntrypointUtils.java:38)
		at java.util.ArrayList.forEach(ArrayList.java:1257)
		at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.logErrors(EntrypointUtils.java:36)
		... 19 more
	Caused by: java.lang.ClassNotFoundException: io.github.cottonmc.cotton.config.ConfigManager
		at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
		at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
		at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:161)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
		... 24 more
```

**Comment by tyra314** [comment]
This should be fixed in 5.1.1-fabric now.
If cotton isn't installed, it now fires this instead of the arcane ClassNotFoundException:
```
Caused by: net.fabricmc.loader.discovery.ModResolutionException: Could not resolve valid mod collection (at: antiqueatlas requires {cotton-config @ [>=1.0.0-rc.1]})
```

Users need to install [Cotton](https://www.curseforge.com/minecraft/mc-mods/cotton) though.
